### ISSUE TITLE
 Error during "git reset --hard origin/master"

### ISSUE THREAD

**Issue by orangejuicy** [Hacklone/private-bower#101]
Probably private-bower believes that every repo have origin/master branch.

```
ESC[32m[bower] ESC[39m ESC[36m19/1/15 10:16:27ESC[39m   Error during "git reset --hard origin/master" in "/home/teamcity/tools/bower/gitRepo
Cache/q"
ESC[32m[bower] ESC[39m ESC[36m19/1/15 10:16:27ESC[39m   ESC[31mCommand failed: fatal: ambiguous argument 'origin/master': unknown revision o
r path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

**Comment by Hacklone** [comment]
You're absolutely right, fixed it :)

**Comment by oliversalzburg** [comment]
This fix conflicts with the solution to #91
